app/serializers/placement_serializer/v01.rb

```diff
@@ -2,18 +2,19 @@
 
 module PlacementSerializer
   class V01
-    EXCLUDE_ATTRIBUTES = %w[
+    EXCLUDED_ATTRIBUTES = %w[
       id
       slug
       trainee_id
+      school_id
```

Comment:
I think we should also have `address` in the `EXCLUDED_ATTRIBUTES` because it seems that this is never populated for Placements, so we'll always just be returning `"address": null`.